tutorials/ph-ug-temp/index.md

```diff
@@ -0,0 +1,106 @@
+
+1. Pull in a shape file of Uganda. 
+
+In these lines of code, we are creating a new variable called "region". However, now we are pulling in a `FeatureCollection` object, and filtering by ‘Country’ to select ‘Uganda.’ FeatureCollections are groups of features (spatial data and attributes).  `Filter` is the command to extract a specific set of feature data from a feature collection. We then map it.
+  ```
+// Import vector file with country boundaries from Earth Engine
+var dataset = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017'); // All countries
+var uganda_border = dataset.filter(ee.Filter.eq("country_na","Uganda")); 
+// apply filter where "country names" equals "Uganda"
+print(uganda_border) 
+// print to ensure that new "uganda_border" object just contains polygon for Uganda
+
+//Add region outline to layer - for selected countries
+Map.addLayer(uganda_border)
+  ```
+2. Read in land surface data.
+
+Now, we need to read in land surface temperature data. `ImageCollection` objects store collections of images. Using the Earth Engine Data Catalog, we can find different image collections. The `ImageCollection` we are reading in here contains the LST data, but we can read in a different `ImageCollection` for other types of data (e.g. vegetation index, or rain fall, light at night).
+Our code applies filters to restrict images for Uganda, within January to December 2015.
+```
+var modis = ee.ImageCollection('MODIS/MOD11A2'); // read in image collection
+var start = ee.Date('2015-01-01'); // set start date
+var dateRange = ee.DateRange(start, start.advance(1, 'year'));
+// set end date 1 year
+var mod11a2 = modis.filterDate(dateRange).filterBounds(uganda_border);
+// apply filter on date range, uganda boundary
+
+var modLSTday = mod11a2.select('LST_Day_1km'); // pull just 1km day LST
+```
+3. Create a function to convert temperature units.
+
+We then need to convert temperature from Kelvin to Celsius. Note that for remote sensing data, sometimes equations required to convert between different units require additional scaling factors, so we should always make sure to check the [documentation](https://icess.eri.ucsb.edu/modis/LstUsrGuide/usrguide_mod11.html#sds) for the specific product we are using when doing conversions.   After referring to the documentation, we learn that in addition to subtracting 273.15, we must multiply the Kelvin value in our MODIS data by a factor of 0.02.
+
+In addition, we will want to keep information about the start and end time for each image in the collection so that we can plot the time series.
+
+We can do both steps at the same time by writing a function to do the conversion, keeping the start and end times using the `copyProperties` function, and then by mapping over the `ImageCollection`. Running this code will generate a new `ImageCollection` called 'modLSTc' (MODIS LST converted) with LST in Celsius, along with the specific time periods we need for the chart.
```

Comment:
Style nit: Please use "you" instead of "we" throughout, and update wording accordingly.
https://developers.google.com/style/highlights#language-and-grammar

Reply:
Done